prow/plank/README.md

```diff
@@ -0,0 +1,41 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+The following job template is based on the format for `spyglass`.
+
+```yaml
+# config.yaml
+
+plank:
+  allow_cancellations: true
+  job_url_prefix: 'https://<domain>/view/gcs/<bucket-name>'
```

Comment:
```yaml
   job_url_prefix: 'https://<domain>/view/gcs/<bucket-name>' # used to link to job results for decorated jobs
```

Reply:
is this url correct though?
going to that link will just show a blank spyglass page without any outputs, also the `<bucket-name>` part might mess up the URL?

Reply:
for example when i used that prefix, it became like
```
https://prow.internal.entelo.com/view/gcs/entelo-ci-test/entelo-ci-test/logs/test-postsubmit/1100260808006307840
```
and `entelo-ci-test` which is the bucket name repeated twice.

Reply:
Yeah you should not need bucket name in there, let your testing be the guide here

Reply:
just using `https://<domain>/view/gcs` for the prefix seems to be fine so maybe i'll just stick with that for the moment.